corehq/apps/hqmedia/models.py

```diff
@@ -537,13 +537,13 @@ def _add_menu_media(item, **kwargs):
             }
             _add_menu_media(module, **media_kwargs)
 
-
-            if (module.case_details.short.lookup_enabled and module.case_details.short.lookup_image):
-                media.append(ApplicationMediaReference(
-                    module.case_details.short.lookup_image,
-                    media_class=CommCareImage,
-                    **media_kwargs)
-                )
+            for _, details, display in module.get_details():
```

Comment:
wasn't blocking, but this seems like a possibly legitimate lint error to care about (if you tried to translate something with `_` later in the function it would fail)